src/MessageProperty/EncryptionInspector.cs

```diff
@@ -87,7 +87,7 @@ bool IsEncryptedMember(MemberInfo arg)
                 }
 
                 // don't try to recurse over members of WireEncryptedString
-                if (member.DeclaringType == typeof(EncryptedString) || member.DeclaringType == typeof(WireEncryptedString))
```

Comment:
so what happens if this code processes a message assembly that targets NSB v6 and still uses `WireEncryptedString`? shouldnt we throw an exception?